__tests__/app.test.js

```diff
@@ -1,3 +1,32 @@
-describe("Name of the group", () => {
-  test("should ", () => {});
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const allData = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(allData);
+});
+
+describe("GET", () => {
+  test("/api/categories: should return array of categories with slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+
+        expect(Array.isArray(body.categories)).toBe(true);
+        expect(body.categories.length > 0).toBe(true);
+
+        body.categories.forEach((category) => {
+          expect(typeof category.slug).toBe("string");
+          expect(typeof category.description).toBe("string");
+        });
+      });

```

Comment:
Good test!

We will need to add another test (at least) to look at errors which we might get with this request.  What happens if I wrote /api/categorie5   ?